docs/book/how-to/stack-deployment/README.md

```diff
@@ -1,19 +1,72 @@
 ---
-description: Stacks are the configuration of your infrastructure.
+description: >
+  Stacks represent the infrastructure and tooling that defines where and how a pipeline executes.
 ---
 
 # Managing stacks & components
 
+
+## What is a stack?
+
 The [stack](../../user-guide/production-guide/understand-stacks.md) is a fundamental component of the ZenML framework. Put simply, a stack represents the configuration of the infrastructure and tooling that defines where and how a pipeline executes.
 
-However, deploying and managing a MLOps stack is tricky 😭😵‍💫. It is not trivial to set up all the different tools that you might need for your pipeline.
+A stack comprises different stack components, where each component is responsible for a specific task. For example, a stack might have a [container registry](../../component-guide/container-registries/container-registries.md), a [Kubernetes cluster](../../component-guide/orchestrators/kubernetes.md) as an [orchestrator](../../component-guide/orchestrators/orchestrators.md), an [experiment tracker](../../component-guide/experiment-trackers/experiment-trackers.md) like MLflow and so on.
```

Comment:
"a Kubernetes cluster" sounds odd here. Maybe you meant to say artifact store?

Reply:
What I have in the text is: a Kubernetes orchestrator as an orchestrator so it would make sense. but you're right, I'll also add the artifact store.